# Action describes a command executable on the robot and its parameters

# Constants defining the type of command
string type
string[] parameters
